### DISCUSSION TITLE
is basic material + fresnel "cheaper" than phong (and why is it not working as expected)

### DISCUSSION THREAD

**Discussion by MarcVdH** [cables-gl/cables_docs#605] [Answers & Questions]
is using basic material + fresnel considerably "cheaper" than using a phong material? (I am not sure about what i see in the performance profiler).
the fresnel seems to not behave as expected in my example patch, or does it?

https://cables.gl/edit/Dv97iL

**Comment by marioepsley** [comment]
Personally i find the fresnel more expensive, but certainly worth testing, I don't know for certain.
You have a white texture so I don't think you'll see the fresnel but it does look like its working correctly when you do this.
<img width="1291" alt="Screenshot 2023-12-30 at 09 55 15" src="https://github.com/cables-gl/cables_docs/assets/36418408/c90f004a-4bbe-44b9-b691-7a11ff1b2f75">
